docs/index.md

```diff
@@ -31,8 +31,11 @@
 <p align="right"><a href="#top">Top</a></p>
 
 ## dictation_asr.proto
-Modified by Techmo, copyright by Google.
-Extensions by Techmo are marked with [**Extension by Techmo**] tag.
+Modified by Techmo, copyright by Google. Changes include:
+- removal of `LongRunningRecognize` support (commented out),
+- modifications of comments, according to how recognition is performed by Techmo,
+- additions that introduce new features to the original API.
+Extensions (parts that were added to the original document) by Techmo are marked with [**Extension by Techmo**] tag.
```

Comment:
Please change items order: first we should mention additions, while 'removal' should be at the end of this list. Additionally all extensions could be listed here. It'll be easier to find it in doc.

Reply:
And you can add also disclaimer, that most of the modifications are: [**Unused**] marks